recipes/element-coder/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "element-coder" %}
+{% set version = "0.0.6" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/element_coder-{{ version }}.tar.gz
+  sha256: 3704c66ca4a2ad16c449f33bdee0112650e391200465774014b77eb11589d944
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - element_coder.encode = element_coder.cli:encode_cli
+    - element_coder.decode = element_coder.cli:decode_cli
+
+
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - python >=3.7
+    - pymatgen
+    - loguru
+    - click
+
+test:
+  imports:
+    - element_coder
+  commands:
+    - pip check
```

```suggestion
    - pip check
    - element_coder.encode --help
    - element_coder.decode --help
```
This checks the CLI. Depends on whether you are actually offering a help function though.